### PR TITLE
Add a hover style for the disabled accounts

### PR DESCRIPTION
**Pull request by Badlapje**
Prior to this change there was no hover style for disabled accounts.

This change adds one.  Testable with this [url in your local environment](https://engine.vm.openconext.org/functional-testing/wayf?connectedIdps=5&displayUnconnectedIdpsWayf=true&unconnectedIdps=5&showIdpBanner=false)

[Issue discovered as part of fixing tickets for the latest feedback round](https://wiki.surfnet.nl/display/coininfra/Bevindingen+nav+tests+door+supportteam)

### CODE REVIEW

File: theme/skeune/stylesheets/mixins-skeune.scss [hunk 1]
```diff
@@ -158,7 +158,11 @@
...
...8V3C15.1711 2.448 14.7359 2 14.1996 2C13.6633 2 13.2281 2.448 13.2281 3V12C13.2281 12.552 12.7929 13 12.2566 13C11.7204 13 11.2851 12.552 11.2851 12V5C11.2851 4.448 10.8499 4 10.3136 4C9.77738 4 9.34216 4.448 9.34216 5V18C9.34216 18.3813 9.12584 18.736 8.79165 18.9013C8.66212 18.9653 8.51575 19 8.37197 19C8.15176 19 7.93674 18.9227 7.76576 18.7813L4.60648 16.1813C4.42255 16.0293 4.20364 15.9507 3.97178 15.9507C3.93292 15.9507 3.89406 15.9533 3.8552 15.9573C3.58318 15.988 3.33966 16.1267 3.16998 16.3453C3.15055 16.3693 3.13371 16.3933 3.11817 16.4187C2.8876 16.7733 2.8876 17.2307 3.11817 17.5853L4.97176 20.444C5.26839 20.9027 5.14792 21.5253 4.70363 21.8307C4.54301 21.9413 4.35649 22 4.16349 22Z' fill='%23EFEFEF'/%3E%3Cpath d='M1.00024 30C0.729562 30 0.466887 29.888 0.278881 29.6933C0.0935417 29.5013 -0.00512807 29.248 0.000205428 28.9813C0.00553893 28.7147 0.113542 28.4653 0.306882 28.28L27.3077 2.28C27.4957 2.1 27.7411 2 28.0011 2C28.2718 2 28.5344 2.112 28.7211 2.30667C28.9065 2.49867 29.0051 2.752 28.9998 3.01867C28.9945 3.28533 28.8865 3.53467 28.6945 3.72L1.69359 29.72C1.50692 29.9013 1.26025 30 1.00024 30Z' fill='%23EFEFEF'/%3E%3C/svg%3E%0A");    background-repeat: no-repeat;
```
Comment (pablothedude) [inline theme/skeune/stylesheets/mixins-skeune.scss:161]: Are those inline svg's really needed?

**Review by pablothedude — APPROVED** [review]

File: theme/skeune/stylesheets/mixins-skeune.scss (hunk 1 above)
Comment (Badlapje) [reply theme/skeune/stylesheets/mixins-skeune.scss:161]: Not really needed, but they are easier to style & make accessible.

File: theme/base/stylesheets/pages/wayf/idpList.scss [hunk 2]
```diff
@@ -160,6 +160,26 @@
             background-color: $softBorderGray;
             box-shadow: 0 1px 0 2px $noAccessGray;
 
+            &:hover {
+                background-color: $lightPink;
+
+                > .idp__content {
+                    > button {
+                        @include button-reset($lightPink);
```
[the remark this answers is not in the document]
Comment (Badlapje) [reply theme/base/stylesheets/pages/wayf/idpList.scss:168]: Normally i'd agree, but for colours that ends up causing a load of extra variables which all refer to the same colour.  So i tend to stick with the names that are used in the stylesheet of the designer.

[the remark this answers is not in the document]
Comment (pablothedude) [reply theme/base/stylesheets/pages/wayf/idpList.scss:168]: I think that if a variable has the same value it doesn't mean they are the same, so it seems a unnecessary abstraction to me. It's nitpicking and as stated earlier, feel free to leave as-is!

**Pull request merged**